src/server/game/DungeonFinding/LFGGroupData.cpp

```diff
@@ -22,19 +22,31 @@
 namespace lfg
 {
     LfgGroupData::LfgGroupData(): m_State(LFG_STATE_NONE), m_OldState(LFG_STATE_NONE),
-        m_Dungeon(0), m_KicksLeft(sWorld->getIntConfig(CONFIG_LFG_MAX_KICK_COUNT))
+        m_Dungeon(0), m_KicksLeft(sWorld->getIntConfig(CONFIG_LFG_MAX_KICK_COUNT)), _isLFGGroup(false)
```

Comment:
![image](https://user-images.githubusercontent.com/16887899/149987795-ee7ef55d-58a2-4c30-a936-709bd1e58d08.png)

Reply:
i guess re-arrange the order.